pkg/transport/anonymizing_serializer.go

```diff
@@ -7,7 +7,7 @@ import (
 
 type AESAnonymizingSerializer[T any] struct {
 	serializer EventSerializer[T]
-	anonymizer pii.StructAnonymizer[aggregate.ID]
+	anonymizer pii.StructAnonymizer[aggregate.Event[T], aggregate.ID]
```

Comment:
I'm wondering whether `pii` should be within this lib, or could it warrant a whole another lib? 🤔

Reply:
yep, eventually it would be best to move it out
I'd like to polish the API a bit first

Reply:
> polish the API a bit first
:flag-pl: